src/containers/counter/counter.container.tsx

```diff
@@ -32,14 +35,55 @@ export class CounterContainer extends Component<CounterContainerProps & CounterC
         const { count, onIncrement } = this.props;
 
         return (
-            <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center', padding: '100px' }}>
-                <div>{count}</div>
-                <div>
-                    <Button onClick={this.onIncrement}>Increment</Button>
+            <Fragment>
+                <div style={{ display: 'flex', justifyContent: 'center' }}>
+                    <Link to="/counter/increment">Increment</Link>
+                    <div style={{ margin: '8px' }} />
+                    <Link to="/counter/decrement">Decrement</Link>
                 </div>
-            </div>
+                <Switch>
+                    <Route
+                        path="/counter/increment"
```

Comment:
This should be replaced by a util or const in the future, wanna do something like `/${ROUTES.COUNTER}/${ROUTES.INCREMENT}`